src/Algebra/Morphism/Construct/Terminal.agda

```diff
@@ -0,0 +1,146 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- The unique morphism to the terminal object,
+-- for each of the relevant categories. Since
+-- each terminal algebra builds on `Monoid`,
+-- possibly with additional (trivial) operations,
+-- satisfying additional properties, it suffices to
+-- define the morphism on the underlying `RawMonoid`
+------------------------------------------------------------------------
+
+{-# OPTIONS --cubical-compatible --safe #-}
+
+open import Level using (Level)
+
+module Algebra.Morphism.Construct.Terminal {c ℓ : Level} where
+
+open import Algebra.Bundles using (Monoid; Group; NearSemiring; Semiring; Ring)
+open import Algebra.Bundles.Raw using (RawMonoid; RawGroup)
+open import Algebra.Construct.Terminal {c} {ℓ}
+import Algebra.Morphism.Definitions as MorphismDefinitions
+open import Algebra.Morphism.Structures
+  using ( module MagmaMorphisms
+        ; module MonoidMorphisms
+        ; module GroupMorphisms
+        ; module NearSemiringMorphisms
+        ; module SemiringMorphisms
+        ; module RingWithoutOneMorphisms
+        ; module RingMorphisms
+        )
+open import Data.Product.Base using (_,_)
+open import Function.Definitions using (StrictlySurjective)
+import Relation.Binary.Morphism.Definitions as Definitions
+open import Relation.Binary.Morphism.Structures
+
+private
+  variable
+    a ℓa : Level
```

Comment:
For better or worse the current standard name is `ℓ` in the library.

Reply:
Yes, but the trouble is, `c` and `ℓ` are already taken by the top-level parametrisation of the module...